kong/clustering.lua

```diff
@@ -351,6 +351,24 @@ local function validate_shared_cert()
   end
 end
 
+local function get_shared_cert_expiration_time()
+  local cert = ngx_var.ssl_client_raw_cert
+
+  if not cert then
+    ngx_log(ngx_ERR, "Data Plane failed to present client certificate " ..
+      "during handshake")
+    return ngx_exit(444)
+  end
+
+  cert = assert(openssl_x509.new(cert, "PEM"))
+  local _, not_after, err = cert:get_lifetime()
```

Comment:
let's use `local not_after, err = cert:get_not_after()` maybe, as it's clearer in function name and we only need the not_after one
```suggestion
  local not_after, err = cert:get_not_after()
```

Reply:
this is a better way definitely! 💯